src/gcp/cloudscheduler.ts

```diff
@@ -108,6 +108,7 @@ export async function createOrReplaceJob(job: Job): Promise<any> {
           )
         );
       }
+      return Promise.reject(err);
```

Comment:
Can you `throw` this error (and the one above it, for that matter) rather than returning a rejected Promise? They should function more or less the same way but we'd be not adding one more Promise to the mix.

```suggestion
      throw err;
```

Reply:
Done!